examples/layers/preprocessing/classification/random_crop_and_zoom.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""random_resized_crop_demo.py.py shows how to use the RandomResizedCrop
+"""random_crop_and_resize.py shows how to use the RandomCropAndResize
```

Comment:
Lets rephrase it as " This demo example shows how to use `RandomResizedCrop`

Reply:
Thank you - let us update the file name to random_crop_and_resize_demo.py

Reply:
Done